### PR TITLE
add bash

### PR DESCRIPTION
**Pull request by f0ti**
added bash and updated on run.sh
Bash is cool mane!

### CODE REVIEW

File: mergesort.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+MergeSort (){
+  local a=($@)
```
Comment (lukasjoc) [inline mergesort.sh:4]: to avoid re-splitting this should be `local a=("$a")`

Comment (lukasjoc) [inline mergesort.sh]: to avoid re-splitting this should be `local a=("$@")`

File: mergesort.sh [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  if [ ${#a[@]} -eq "1" ]
+  then
+    r=("${a[@]}")
+    echo ${r[@]}
```
Comment (lukasjoc) [inline mergesort.sh]: to avoid re-splitting this should be `echo "${r[@]}"`

File: mergesort.sh [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  elif [[ "${#a[@]}" -eq "2" ]]
+  then
+    if [[ "${a[0]}" -gt "${a[1]}" ]]
+    then
+      local t=("${a[1]}" "${a[0]}")
+      echo ${t[@]}
+    else
+      r=("${a[@]}")
+      echo ${r[@]}
```
Comment (lukasjoc) [inline mergesort.sh]: to avoid re-splitting this should be `echo "${r[@]}"`

File: mergesort.sh [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  if [ ${#a[@]} -eq "1" ]
+  then
+    r=("${a[@]}")
+    echo ${r[@]}
+  elif [[ "${#a[@]}" -eq "2" ]]
+  then
+    if [[ "${a[0]}" -gt "${a[1]}" ]]
+    then
+      local t=("${a[1]}" "${a[0]}")
+      echo ${t[@]}
```
Comment (lukasjoc) [inline mergesort.sh]: to avoid re-splitting this should be `echo "${t[@]}"`

File: mergesort.sh [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+        unset m1
+      elif [ "${#m2[@]}" -gt 0 ]
+      then
+        ret=("${ret[@]}" "${m2[@]}")
+        unset m2
+      else
+        break
+      fi
+    done
+    echo ${ret[@]}
```
Comment (lukasjoc) [inline mergesort.sh]: to avoid re-splitting this should be `echo "${ret[@]}"`

File: mergesort.sh [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+  fi
+}
+
+a=(5 9 1 3 4 6 6 3 2)
+echo $(MergeSort ${a[@]})
```
Comment (lukasjoc) [inline mergesort.sh]: You do not even need the `echo` here `MergeSort "${a[@]}"` will do it

**Comment by f0ti** [comment]
Made all the changes. Thanks bro. Anything else to change before I make a new PR?

**Comment by monksy** [comment]
Update the readme

**Pull request merged**